auth-service/src/main/kotlin/uk/gov/communities/delta/auth/utils/UserService.kt

```diff
@@ -0,0 +1,153 @@
+package uk.gov.communities.delta.auth.utils
+
+import java.util.*
+
+class UserService {
+//    TODO - implement any needed functions
+//              - findUserById
+//              - createUser
+//              - any they (^) use
+
+    //    private val logger = LoggerFactory.getLogger(javaClass)
+//
+//    fun findUserById(id: String): Optional<User> {
+//        return findUserById(id, false)
+//    }
+//
+    fun findUserById(id: String?, includeNonDeltaUsers: Boolean): Optional<User> {
```

Comment:
Delta wasn't, but we should be explicit, is this CN or email we're searching by?